### DISCUSSION TITLE
What CSS reset do you prefer?

### DISCUSSION THREAD

**Discussion by louwers** [unocss/unocss#3008] [General]
There are various options? https://unocss.dev/guide/style-reset

Which do you prefer to use with UnoCSS?

**Comment by chz** [comment]
If you used tailwind before i recommend use @unocss/reset/tailwind.css

else Eric Meyer or normalize

**Reply by lizyChy0329** [reply]
[tailwind-compat](https://unocss.dev/guide/style-reset#tailwind-compat) is a better choice.

**Comment by ntnyq** [comment]
I use `@unocss/reset/tailwind.css` mostly.
